README.md

```diff
@@ -48,11 +48,21 @@ and you need to setup them into `.env`, see `example.env`.
 Then open console and run
 
 ```shell
-hardhat node
+npm run test
 ```
 
-Open another console and run
+You should get result like this:
 
 ```shell
-npm run test
+  Umbrella First Class Data - Layer 1
+Deploying YourContract with Umbrella Chain: 0x10816ea91c8001b3a3890e8c597c5972b4e4abfc
+    getting latest numeric FCD for BTC-USD
```

Comment:
I thought we were getting rid of FCD...

Reply:
I do not remember such a decision...

Reply:
we planned to rename it to
On-chain data, L2 data as far as I remember